### ISSUE TITLE
Azure Machine Learning Support?

### ISSUE THREAD

**Issue by drewB** [langchain-ai/langchainjs#2872]
Is there a way to use Azure Machine Learning with js LangChain? Specifically, want to use a llama2 model. Looks like the python version supports AML - https://python.langchain.com/docs/integrations/llms/azure_ml

If there is no current way, could you point me toward creating my own connector?

**Comment by jacoblee93** [comment]
This would be awesome - you can have a look here for our contribution guidelines for integrations:

https://github.com/langchain-ai/langchainjs/blob/main/CONTRIBUTING.md#want-to-add-a-specific-integration

**Comment by drewB** [comment]
I have not started and likely wouldn't be able to for a while.  Please feel free to take it on.

**Comment by Ultrahi** [comment]
Alright, thank you. We will happily take it on.

**Comment by iserifith** [comment]
@Ultrahi are these still ongoing?

**Comment by Ultrahi** [comment]
Yes, my university colleagues and I are still working on it. We are planning to push out a pr before the end of the month.

**Comment by drewB** [comment]
Not sure if you all saw this announcement from Microsoft but seems like this new pay-as-you-go model has a more simple api and cost structure.

https://techcommunity.microsoft.com/t5/ai-machine-learning-blog/announcing-llama-2-inference-apis-and-hosted-fine-tuning-through/ba-p/3979227

**Comment by drewB** [comment]
I think fine to close since there is activity on the PR so it can be taken forward from there.
